node/runtime/src/lib.rs

```diff
@@ -407,6 +407,9 @@ impl offences::Trait for Runtime {
 	type OnOffenceHandler = Staking;
 }
 
+impl authority_discovery::Trait for Runtime {
+}
```

Comment:
Move this up.

Reply:
I am guessing you want the `xxx::Trait` implementations to be alphabetical @bkchr?

Reply:
No just the `}`. I was reviewing on mobile, so I could not provide a suggestion.